api-participant/src/main/resources/api/openapi.yml

```diff
@@ -231,6 +231,22 @@ paths:
           content: { application/json: { schema: { type: object } } }
         '500':
           $ref: '#/components/responses/ServerError'
+  /api/portals/v1/{portalShortcode}/env/{envName}/studies/{studyShortcode}/enrollee/{enrolleeShortcode}/outreach:
+    get:
+      summary: List the outreach activities available to the signed-in user
+      tags: [ outreach ]
+      operationId: listOutreachActivities
+      parameters:
+        - { name: portalShortcode, in: path, required: true, schema: { type: string } }
+        - { name: envName, in: path, required: true, schema: { type: string } }
+        - { name: studyShortcode, in: path, required: true, schema: { type: string } }
```

Comment:
I'm realizing that study shortcode isn't used anywhere, and I'm wondering if that's right or if I'm missing something.